### PR TITLE
chore(lint): remove unused imports

### PR DESCRIPTION
**Pull request by alexeagle**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

This was done automatically by tslint, which can now fix issues it finds.
The fixer is still pending in PR https://github.com/palantir/tslint/pull/1568
Also I have a local bugfix for https://github.com/palantir/tslint/issues/1569
which causes too many imports to be deleted.

### CODE REVIEW

File: modules/@angular/benchpress/test/metric/multi_metric_spec.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {AsyncTestCompleter, afterEach, beforeEach, ddescribe, describe, expect, iit, inject, it, xit} from '@angular/core/testing/testing_internal';
+import {AsyncTestCompleter, describe, expect, inject, it} from '@angular/core/testing/testing_internal';
```
Comment (vicb) [inline modules/@angular/benchpress/test/metric/multi_metric_spec.ts]: could we remove ` describe, expect, inject, it` here please (ok if follow up PR)

**Review by vicb — COMMENTED** [review]
1 comment, ok if part of a follow up PR
please fix the CI

Great PR, thanks !

File: modules/@angular/benchpress/test/metric/multi_metric_spec.ts (hunk 1 above)
Comment (alexeagle) [inline modules/@angular/benchpress/test/metric/multi_metric_spec.ts]: let's discuss as follow-up. this is just the automated, reproducible change.

**Comment by rkirov** [comment]
needs rebase

**Comment by alexeagle** [comment]
rebased

**Pull request merged**